web/components/feed/feed-items.tsx

```diff
@@ -118,24 +118,80 @@ function FeedItem(props: { item: ActivityItem }) {
       return <FeedResolve {...item} />
     case 'commentInput':
       return <CommentInput {...item} />
+    case 'commentThread':
+      return <FeedCommentThread {...item} />
   }
 }
 
+export function FeedCommentThread(props: {
+  contract: Contract
+  comments: Comment[]
+  betsBySameUser: Bet[]
+  truncate?: boolean
+  smallAvatar?: boolean
+}) {
+  const { contract, comments, betsBySameUser, truncate, smallAvatar } = props
+  const [showCommentReplyInput, setShowCommentReplyInput] = useState(false)
+  const [replyToUsername, setReplyToUsername] = useState('')
+  const router = useRouter()
+  const scrollToId = '#' + comments[comments.length - 1].id
+  function scrollAndOpenReplyInput(comment: Comment) {
```

Comment:
Use useState to set the ref to the textarea HTMLTextAreaElement, and set focus here